tests/harvest/test_harvest_cli.py

```
@@ -4,31 +1,41 @@
+    "args_line, expected",
+    [
+        (
+            "-c /path/to/config -d /path/to/dump",
+            {
+                "config": ("/path/to/config",),
+                "dump": ("/path/to/dump",),
+            },
+        ),
+    ],
+)
+def test_args(args_line, expected) -> None:
```

Ruff

Missing type annotation for function argument `args_line`

(ANN001)